README.md

```diff
@@ -107,6 +108,29 @@ When `--token your-token` is provided, an authentication header is required. For
 ~$ curl localhost:8980/transactions -H "X-Indexer-API-Token: your-token"
 ```
 
+# Settings
+
+Settings can be provided from the command line, a configuration file, or an environment variable
+
+### daemon mode
+
+```
+./algorand-indexer daemon [options]
+```
+
+| Command Line Flag (long) | (short) | Config File                | Environment Variable               |
+| ------------------------ | ------- | -------------------------- | ---------------------------------- |
+| postgres                 | P       | postgres-conenction-string | INDEXER_POSTGRES_CONNECTION_STRING |
+| pidfile                  |         | pidfile                    | INDEXER_PIDFILE                    |
+| algod                    | d       | algod-data-dir             | INDEXER_ALGOD_DATA_DIR             |
```

Comment:
this could be just ALGORAND_DATA same as for `algod`,
and I see that code is still in there, so maybe this is just a doc change?

Reply:
Yeah I wasn't sure how to deal with that. I didn't want to remove it so currently it's acting as the final fallback if it wasn't provided anywhere else.

Reply:
In my usage I prefer to set ALGORAND_DATA once and have it work for both `goal` and `algorand-indexer`.
Probably in a proper production system it should always be explicitly set, but this way is handy for development.

Reply:
Yeah that seems reasonable, I'm not planning to remove ALGORAND_DATA